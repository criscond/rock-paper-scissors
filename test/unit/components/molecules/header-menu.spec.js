import headerMenu from '../../../../src/js/components/molecules/header-menu';
describe('Header Menu', () => {

	it('should create a menu', () => {
		expect(headerMenu()).toMatchSnapshot();
	});
});
