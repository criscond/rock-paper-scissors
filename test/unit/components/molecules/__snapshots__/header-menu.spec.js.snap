// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Menu should create a menu 1`] = `
<ul>
  <li>
    <a
      href="https://github.com/criscond/rock-paper-scissors"
      rel="noreferrer"
      target="_blank"
      title="The GitHub page of the project"
    >
      GitHub
    </a>
  </li>
  <li>
    <a
      href="https://en.wikipedia.org/wiki/Rock-paper-scissors"
      rel="noreferrer"
      target="_blank"
      title="The WiKi page of the project"
    >
      WiKi
    </a>
  </li>
</ul>
`;
